plotly/tools.py

```diff
@@ -2275,6 +2277,25 @@ def create_candlestick(open, high, low, close,
         return dict(data=data, layout=layout)
 
 
+    @staticmethod
+    def create_dendrogram(X, orientation="bottom", labels=None,
+                          colorscale=None, **kwargs):
```

Comment:
Any use for the `**kwargs`? It doesn't look like it's been used.

We *may* want to allow updates like these, but maybe we should hold off for now.